### DISCUSSION TITLE
Optional namespacing of Tabs fields

### DISCUSSION THREAD

**Discussion by slavanossar** [payloadcms/payload#776] [Feature Requests & Ideas]
I've been playing around with the new `Tabs` field, really nice addition (congrats on v1!). I was previously using the `Group` field to separate lots of fields logical groupings. This also allowed reusing fields in multiple groups on a collection without worrying about name collisions, since all fields within a group were contained within their own object based on the Group `name`.

Since the `Tabs` field doesn't do this, I'm currently placing a single `Group` field within the tab so I can retain the existing field structure.

I understand the `Tabs` field is designed to be purely presentational, but it might be useful if tabs also had an optional `name` field that made them function similarly to a `Group` field? It would give more flexibility for how they are used, and eliminate the need for a single redundant `Group` field to namespace the fields within.

**Comment by DanRibbens** [comment]
We love this idea!
It makes sense for `tabs`, `groups`, and `collapsible` fields to have an optional `name` property. If it is on a field, it affects the shape of the document in storage, if not the field is used for the UI only. 
A PR could help bring this in faster, we've got a few other priorities atm.

Thanks!

**Reply by slavanossar** [reply]
Cool, I've had a go at implementing this. I created a draft PR, got a couple of questions about it: https://github.com/payloadcms/payload/pull/814
